### DISCUSSION TITLE
No caching for the hosted version of Plausible

### DISCUSSION THREAD

**Discussion by Scooletz** [plausible/analytics#1305] [Anything Else]
I recently installed a Cloudflare Worker on a site to ensure that no blocker removes the analytics from my site. To make that happen I followed the guidelines https://plausible.io/docs/proxy/guides/cloudflare . The snippet in there uses the Cloudflare's `cache` to cache a response from the origin. I noticed though that it's never hit and I always receive the `BYPASS` status. I dug deeper and found that he origin `https://plausible.io/js/plausible.js` returns the response with the following headers

```
access-control-allow-origin
	*
age
	1
cache-control
	max-age=0, private, must-revalidate
content-encoding
	br
content-length
	639
content-type
	application/javascript
cross-origin-resource-policy
	cross-origin
date
	Tue, 07 Sep 2021 16:58:03 GMT
server
	Netlify
vary
	Accept-Encoding
X-Firefox-Spdy
	h2
```

making it uncacheable. Have you considered adding ETag at least (with the revalidate) or add some caching? It'd be awesome to be able to use the Cloudflare's caching capabilities.

**Comment by ukutaht** [comment]
Hey thanks for reporting.

We have to use these cache headers because Netlify. See:

https://github.com/plausible/docs/issues/121
https://github.com/plausible/analytics/issues/1085#issuecomment-849694782

**Comment by metmarkosaric** [comment]
Our script is now being cached so there should not be a need for any changes anymore

**Comment by Scooletz** [comment]
Just verified and saw an awesome `304 Not Modified`. Thank you 🤗 

![image](https://user-images.githubusercontent.com/519707/150761764-cab63bf4-4b59-4d58-8107-e1cdc18f47cc.png)

**Reply by metmarkosaric** [reply]
nice, thanks for the update!
